src/components/MedicationGroup.js

```diff
@@ -22,13 +33,26 @@ export default function MedicationGroup({ id, time, meds, handleDeleteClick }) {
           <li key={id}>{medName}</li>
         ))}
       </Meds>
-      <SmallButton
-        right="10px"
-        top="10px"
-        onClick={() => handleDeleteClick(id)}
-      >
-        -
-      </SmallButton>
+      {editMode && (
+        <>
+          <SmallButton
+            right="10px"
+            top="10px"
+            color="red"
+            onClick={() => handleDeleteClick(id)}
+          >
+            <img src={deleteIcon} alt="" height="16px" />
+          </SmallButton>
+          <SmallButton
+            right="10px"
+            top="40px"
+            color="green"
+            onClick={() => handleEditClick(id)}
+          >
+            <img src={editIcon} alt="" height="14px" />
+          </SmallButton>
+        </>
+      )}
```

Comment:
I really like the idea of an edit mode which triggers a different layout with buttons! 🚀